themes/src/main/resources-community/theme/base/login/messages/messages_de.properties

```diff
@@ -336,6 +336,7 @@ requiredAction.UPDATE_PROFILE=Profil aktualisieren
 requiredAction.VERIFY_EMAIL=E-Mail-Adresse verifizieren
 requiredAction.CONFIGURE_RECOVERY_AUTHN_CODES=Wiederherstellungscodes einrichten
 requiredAction.webauthn-register-passwordless=Kennwortlose Authentifizierung einrichten
+requiredAction.webauthn-register=Sicherheitsschlüssel einrichten
```

Comment:
German translations are managed via https://hosted.weblate.org/projects/keycloak/

Please remove the German translation here and update the translations via the website once this PR has been merged and the English key is present in the message file. Otherwise we might see hickups/conflicts in Weblate, which are a pain to resolve. Thanks!

```suggestion
```

Reply:
Removed the german translation. I couldn't find the key in weblate, does it only show up after the english translation is added?

Reply:
Eactly.